tools/travis.ps1

```diff
@@ -235,7 +234,7 @@ elseif($Stage -eq 'Build')
     $pesterPassThruNoSudoObject = Start-PSPester @pesterParam
 
     # Running tests, which require sudo.
-    $pesterParam['Tag'] = @('RequireSudoOnUnix')
+    $pesterParam['Tag'] += 'RequireSudoOnUnix'
```

Comment:
In the case where this is a full build, the effective pester params will be @('CI','Feature','Scenario',RequireSudoOnUnix').

This is not correct since we have already executed tests which do not require Sudo.